recipes/sparc-x-api/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "SPARC-X-API" %}
+{% set version = "1.0.0" %}
+{% set file_ext = ".tar.gz" %}
+
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/SPARC-X/{{ name }}/archive/refs/tags/v{{ version }}{{ file_ext }}
+  sha256: 34aa3a8d0801a91736b0d8322d78d1e432e92585e61af54db3aba17340ff7617
+
+build:
+  noarch: python
+  number: 0
+  # Install and include the psp files
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv && cd .. && {{ PYTHON }} -m sparc.download_data"
+  entry_points:
+    - "sparc-ase = sparc.cli:main"
+    
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools
+    - ase >=3.22
+    - pip
+  host:
+    - python >=3.8
+    - ase >=3.22
```

Comment:
You don't need runtime dependencies at build time. Unless the package imports itself at build time, then you should report this upstream.

```suggestion
```

Reply:
Indeed the line `script: "{{ PYTHON }} -m pip install . --no-deps -vv && cd .. && {{ PYTHON }} -m sparc.download_data"` uses a post-installation command to add additional binary data which requires imports. is this allowed in conda-forge or I should use alternative approaches?

Reply:
Did you try removing it? If it is post installation you don't need it in build.
Note that conda also has [post link](https://docs.conda.io/projects/conda-build/en/stable/resources/link-scripts.html#adding-pre-link-post-link-and-pre-unlink-scripts) for this kind of thing if just removing the run-time dependency doesn't work.

Reply:
@ocefpaf post-link sounds a better option. I have updated the recipe with the postlink scripts and subsequently changed the tests. appreciate your feedback